src/Map.jl

```diff
@@ -13,11 +13,14 @@
 Map(::Type{T}) where T <: Map = supertype(T)
 Map(::Type{S}) where S <: SetMap = Map{D, C, <:S, T} where {D, C, T}
 
-get_field(M, f) = getfield(M, f) # fall back to Julia builtin
-set_field!(M, f, g) = setfield!(M, f, g) # fall back to Julia builtin
+function domain end
+function codomain end
+function image_fn end
+
+domain(f::Map) = f.domain # fallback, FIXME: to be removed eventually
+codomain(f::Map) = f.codomain # fallback, FIXME: to be removed eventually
```

Comment:
I enabled these fallbacks for `domain` and `codomain`, which I think will fix the `OscarCI` failures. I think it would be better to add these methods explicitly for all types that need it, so e.g. also `Oscar.MPolyAnyMap`. But for now this makes it easier to migrate